src/conv-ccs/conv-ccs.C

```diff
@@ -598,7 +599,12 @@ void CcsInit(char **argv)
      }
 
      if (CmiGetArgFlagDesc(argv, "+DebugSuspend", "Suspend execution at beginning of program")) {
+#if CMK_SMP       
+       if(CmiMyRank() != CmiMyNodeSize()) CpvAccess(cpdSuspendStartup) = 1;
```

Comment:
This block could be:

```
#if CMK_SMP
      if (!CmiInCommThread())
#endif
        CpvAccess(cpdSuspendStartup) = 1;
```

At the least, testing `CmiMyRank()` in non-SMP mode is extraneous.

Reply:
done in fixup